src/testRunner/parallel/worker.ts

```diff
@@ -209,8 +209,14 @@ namespace Harness.Parallel.Worker {
             const errors: ErrorInfo[] = [];
             const passes: TestInfo[] = [];
             const start = +new Date();
-            const runner = new Mocha.Runner(suite, /*delay*/ false);
-
+            /* DeprecationWarning: "Runner(suite: Suite, delay: boolean)" is deprecated. Use "Runner(suite: Suite, {delay: boolean})" instead.*/

```

Comment:
Temporary work around because Mocha Type Definition is not up to date  
 - [Type Definition (8.2) expects: `constructor(suite: Suite, delay: boolean);`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fba3f8bd9d82d6dae7ae6bc56eef52d0432e9f9a/types/mocha/index.d.ts#L1403)
 - [Mocha (9.0.2) expects: `constructor(suite, opts)`](https://github.com/mochajs/mocha/blob/master/lib/runner.js#L143)

Reply:
FWIW I ran `npm i @types/mocha@latest` locally and it was updated to a version that has `RunnerOptions | boolean`. So this workaround shouldn't be necessary anymore @Luxcium 👍

Reply:
Thanks, I'll close this then.